test/istanbultestrunner.ts

```diff
@@ -226,3 +164,65 @@ class CoverageRunner {
     }
 }
 
+function _readCoverOptions(testsRoot: string): ITestRunnerOptions {
```

Comment:
probably should remove the `_` prefix since that isn't our normal coding convention in this project

Reply:
There's a ton of leading underscores and the tslint was set up to specifically allow that so I'm going to leave that one in for now.